user/serializers.py

```diff
@@ -1,24 +1,67 @@
-from django.contrib.auth import get_user_model
+from django.contrib.auth import get_user_model, authenticate
 from rest_framework import serializers
+from django.utils.translation import gettext as _
 
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = get_user_model()
         fields = ("id", "email", "password", "is_staff")
-        read_only_fields = ("is_staff",)
-        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password":
+                {
+                    "write_only": True,
+                    "min_length": 5,
+                    "style": {"inpute_type": "password"},
```

Comment:
There is a typo in the dictionary key 'style'. It should be 'input_type' instead of 'inpute_type'.